pkg/k8s/watchers/pod.go

```diff
@@ -191,6 +191,13 @@ func (k *K8sWatcher) podsInit(k8sClient kubernetes.Interface, asyncControllers *
 	}
 }
 
+func skipHostNetworkedPod(pod *slim_corev1.Pod) bool {
```

Comment:
This function name is a little bit misleading because it's not skipping host network pods. What it is doing is checking if it is a hostnetwork pod **and** LRP is disabled.